llm/codellama/gui.yaml

```
@@ -0,0 +1,39 @@
+envs:
+  ENDPOINT: http://x.x.x.x/3031
```

Can we add a quick comment here about what this ENDPOINT is?
Also, should this contain `http` given L28 already adds `http://` before the `${ENDPOINT}`?
```suggestion
  ENDPOINT: x.x.x.x:3031 # Address of the API server running codellama. 
```

---

Good catch! Added. Thanks!